src/FakeItEasy/CallOrderingContext.cs

```diff
@@ -0,0 +1,98 @@
+namespace FakeItEasy
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using FakeItEasy.Core;
+
+    internal class CallOrderingContext : ICallOrderingContext
+    {
+        private readonly CallWriter callWriter;
+        private readonly HashSet<FakeManager> fakeManagers;
+        private readonly List<AssertedCall> assertedCalls;
+        private int currentSequenceNumber;
+
+        public CallOrderingContext(CallWriter callWriter)

```

Comment:
> made public so Fake.InitializeFixture could access it?

Yes

> Seems odd that it would make a difference on an internal class.

That surprised me too, but apparently it makes a difference for reflection...

> I wonder if there's an opportunity to improve Fake.InitializeFixture.

Maybe there is. It would be easy enough to fix (just add a few `BindingFlags` to the `GetConstructors()` call)